src/test/java/jalau/at18/katas/highestlowest/sarai/MainTest.java

```diff
@@ -0,0 +1,11 @@
+package jalau.at18.katas.highestlowest.sarai;
+
+import static org.junit.Assert.*;
+
+public class MainTest {
+    public void shouldReturnHighestlowestNumbers() {
```

Comment:
Your tests are not running, they are missing the `@Test` annotation, please add it for all your unit tests